backend/inspirehep/records/views.py

```diff
@@ -239,6 +246,32 @@ def reference_self_curation(args):
     return jsonify({"message": "Success"}), 200
 
 
+@blueprint.route("/literature/<path:recid>/<int:old_revision>..<int:new_revision>")
+@login_required_with_roles([Roles.superuser.value, Roles.cataloger.value])
+def reference_diff(recid, old_revision, new_revision):
```

Comment:
something like :) wdyt?
```suggestion
def literature_reference_difference_between_versions(recid, old_revision, new_revision):
```